include/modem_info.h

```diff
@@ -19,37 +19,99 @@
 #define MODEM_INFO_MAX_RESPONSE_SIZE 100
 
 /** Size of the JSON string. */
-#define MODEM_INFO_JSON_STRING_SIZE 256
+#define MODEM_INFO_JSON_STRING_SIZE 512
 
 /** RSRP offset value. */
 #define MODEM_INFO_RSRP_OFFSET_VAL 141
 
+/** Maximum string size of the network mode string */
+#define MODEM_INFO_NETWORK_MODE_MAX_SIZE 12
+
 /**@brief RSRP event handler function protoype. */
 typedef void (*rsrp_cb_t)(char rsrp_value);
 
 /**@brief LTE link information data. */
 enum modem_info {
 	MODEM_INFO_RSRP,	/**< Signal strength. */
-	MODEM_INFO_BAND,	/**< Current LTE band. */
-	MODEM_INFO_MODE,	/**< Current mode. */
+	MODEM_INFO_CUR_BAND,	/**< Current LTE band. */
+	MODEM_INFO_SUP_BAND,	/**< Supported LTE bands. */
+	MODEM_INFO_AREA_CODE,   /**< Tracking area code. */
+	MODEM_INFO_UE_MODE,	/**< Current mode. */
 	MODEM_INFO_OPERATOR,	/**< Current operator name. */
+	MODEM_INFO_MCC,		/**< Mobile country code. */
+	MODEM_INFO_MNC,		/**< Mobile network code. */
 	MODEM_INFO_CELLID,	/**< Cell ID of the device. */
 	MODEM_INFO_IP_ADDRESS,  /**< IP address of the device. */
 	MODEM_INFO_UICC,	/**< UICC state. */
 	MODEM_INFO_BATTERY,	/**< Battery voltage. */
 	MODEM_INFO_TEMP,	/**< Temperature level. */
 	MODEM_INFO_FW_VERSION,  /**< Modem firmware version. */
 	MODEM_INFO_ICCID,	/**< SIM ICCID */
+	MODEM_INFO_LTE_MODE,	/**< LTE-M support mode. */
+	MODEM_INFO_NBIOT_MODE,	/**< NB-IoT support mode. */
+	MODEM_INFO_GPS_MODE,	/**< GPS support mode. */
 	MODEM_INFO_COUNT,	/**< Number of legal elements in the enum. */
 };
 
-/** @brief Initialize the link information driver.
+struct lte_param {
+	u16_t value;
+	char string[MODEM_INFO_MAX_RESPONSE_SIZE];
```

Comment:
`char string` is very ambiguous - could we use a more descriptive name perhaps? Especially there's no docs for this structure.

Reply:
Changed name to `value.string`